bitnami/mxnet/templates/standalone/deployment.yaml

```diff
@@ -0,0 +1,233 @@
+{{- if eq .Values.mode "standalone" }}
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: Deployment
+metadata:
+  name: {{ include "common.names.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: standalone
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "common.labels.matchLabels" . | nindent 6 }}
+      app.kubernetes.io/component: standalone
+  template:
+    metadata:
+      {{- if .Values.standalone.podAnnotations }}
+      annotations: {{- include "common.tplvalues.render" (dict "value" .Values.standalone.podAnnotations "context" $) | nindent 8 }}
+      {{- end }}
+      labels: {{- include "common.labels.standard" . | nindent 8 }}
+        app.kubernetes.io/component: standalone
+        {{- if .Values.standalone.podLabels }}
+        {{- include "common.tplvalues.render" (dict "value" .Values.standalone.podLabels "context" $) | nindent 8 }}
+        {{- end }}
+    spec:
+      {{- include "mxnet.imagePullSecrets" . | nindent 6 }}
+      {{- if .Values.standalone.hostAliases }}
+      hostAliases: {{- include "common.tplvalues.render" (dict "value" .Values.standalone.hostAliases "context" $) | nindent 8 }}
+      {{- end }}
+      {{- if .Values.standalone.updateStrategy }}
+      strategy: {{- toYaml .Values.standalone.updateStrategy | nindent 6 }}
```

Comment:
It should be in `spec.strategy` not in `spec.template.spec.strategy`

Reply:
Nice catch! My bad, I hadn't noticed it was under the wrong spec